git-keeper.py

```diff
@@ -24,63 +24,82 @@
 tar = sh.tar.bake('-cf')
 workdir = 'workdir'
 
+
 def upload2s3(s3_client, repo_tar, git_keeper_bucket, date, object_name):
     try:
-        response = s3_client.upload_file(repo_tar, git_keeper_bucket, date + '/' + object_name)
+        s3_client.upload_file(
+            repo_tar, git_keeper_bucket, date + '/' + object_name)
```

Comment:
I still feel like the try/catch surrounding this is unnecessary. What are we gaining with this try/catch? Without the try/catch the execution will also stop if it raises a ClientError.

Reply:
also, use `os.path.join` for  the last component.

Reply:
I think we want to log error and continue to next repo